lib/log.js

```diff
@@ -132,6 +133,12 @@ Lug.prototype.notifyAttachedServices = function (name, request, data) {
         // Add a timestamp that this event occurred to help attached services resolve any
         // potential timing issues
         data.ts = data.ts || Date.now() / 1000 // Convert to float seconds
+
+        // convert an oauth client-id to a human readable format
+        if (data.service && data.service !== 'sync') {
```

Comment:
I would probably add a little note that this sets human readable format only if a service name exists, else it will keep the current service name.